raiden/transfer/events.py

```diff
@@ -51,6 +51,7 @@ class SendWithdrawRequest(SendMessageEvent):
     participant: Address
     expiration: BlockExpiration
     nonce: Nonce
+    coop_settle: Optional[bool] = False
```

Comment:
Why is this optional?

Reply:
Because not all withdraws are coop settlements. There is still the "normal" withdraw.

Reply:
But what's the third state? 
`Optional[bool]` is tri-state (`True | False | None`).

I don't see `None` being used and also can't think what that would mean in this context.

Reply:
I made it non-optional.